website/docs/reference/warehouse-profiles/athena-profile.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Athena Profile"
+---
+
+## Overview of dbt-athena
+
+**Maintained by:** Community
+**Author:** Tomme
+**Source:** [Github](https://github.com/Tomme/dbt-athena)
+**dbt Cloud:** Not Supported  
+**dbt Slack channel** [Link to channel](https://getdbt.slack.com/archives/C013MLFR7BQ)
+
+The easiest way to install is to use pip:
+
+    pip install git+https://github.com/Tomme/dbt-athena.git
+
+## Connecting to Athena with dbt-athena
+
+This plugin does not accept any credentials directly. Instead, [credentials are determined automatically](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html) based on aws cli/boto3 conventions and stored login info. You can configure the AWS profile name to use via aws_profile_name. Checkout DBT profile configuration below for details.
```

Comment:
Hi! Can you lowercase dbt? It is always lowercase

Reply:
Done :)